### ISSUE TITLE
Project Description text template needed

### ISSUE THREAD

**Issue by clintmclean** [wearethoughtfox/wearethoughtfox-wp#11]
Below is a suggestion of format - open to it being ripped apart. For large projects could we have a series of these that we can click through?

Title of Project

Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project. Brief of project.

Component: Description of component such as design or concept or web build etc. Description of component such as design or concept or web build etc.

Component: Description of component such as design or concept or web build etc. Description of component such as design or concept or web build etc.

Component: Description of component such as design or concept or web build etc. Description of component such as design or concept or web build etc.

Component: Description of component such as design or concept or web build etc. Description of component such as design or concept or web build etc.

**Comment by robertocarroll** [comment]
My idea was to have quite a simple project page and then use our blog to write about particular aspects that are interesting ... Then link up these specific blog posts to the project  .... that's how it is working at the moment ... and I quite like that approach.  

Title 
Year
Type of work [taxonomy]
Summary 
Images [Royal Slider]
Text - three paragraphs max - general description of project.  

Blog posts about project 
1. 
2. 
3. 

It offers the freedom to write lots or not write much at all and it sets a good tone - informal, working in public, interesting and interested.  

So for example, the BBC Connected Project I might write something about companion-centred design 

What do you reckon?

**Comment by clintmclean** [comment]
Hi - yeah that's fine with me as long as all the necessary info is available for potential clients to click through the work and get all the info they need. The blog obviously would add to it but if they want to 'scan' our work I think they wouldn't want to bounce back and forth.

**Comment by UnaJanicijevic** [comment]
I'd love to ad 'fun bonus' stuff to the blog, like the experiments with spray paint I did for the Barjeel logo; or something about the 'Arabic for Designers' book Clint gave me
and on the technical end, any lessons we learned and want to share, etc etc

**Comment by clintmclean** [comment]
A first go. Very rough but wanted to get something on the site.

Barjeel Art Foundation
2013
Logo, Branding, Website design, Website build, online archive, Print catalogue, postcards, Stationary, Business cards

For the Barjeel Art Foundation we did a ground up rebranding and website design and build. We began by replacing their existing logo with a more contemporary one using a unique colour combination. The logo set the tone for the project and anchored wide reaching components from Wall signage, outdoor banners, the website, catalogue and canvas bags given out at the gallery.

The online identity for Barjeel is one part website and one part archive of the collection. Central to our concept was to have the collection searchable by a variety of criteria including country, artist, medium and personally curated sections.

We'd like to thank Mandy and Sultan at Barjeel for having faith in our ideas for this project.
